airbyte-integrations/connector-templates/python-source/template_python_source/__init__.py

```diff
@@ -22,4 +22,6 @@
 SOFTWARE.
 """
 
-from .source import *
+from .source import TemplatePythonSource
+
+__all__ = ["TemplatePythonSource"]
```

Comment:
For my own curiosity, why do we need that line?

Reply:
> To better support introspection, modules should explicitly declare the names in their public API using the __all__ attribute.

Moreover, it solves `flake8` warnings:
https://stackoverflow.com/a/49266468